src/components/ComparisonMap/ComparisonMap.vue

```diff
@@ -159,14 +160,16 @@
         // On mounted, `date` is the last date in the time series
         // `oldDate` is the closest date in the time series one year before `date`
         const date = this.timeSeriesDates[this.timeSeriesDates.length - 1]
+        this.isoDate = date.toISOString().substring(0, 10)
```

Comment:
You can also use a custom format.

Reply:
For a custom format I would [need a library](https://stackoverflow.com/questions/1056728/where-can-i-find-documentation-on-formatting-a-date-in-javascript) like Moment.js